src/Form/CitizenProjectCommandType.php

```diff
@@ -31,6 +32,7 @@ public function __construct(CitizenProjectSkillTransformer $citizenProjectSkillT
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $command = $builder->getData();
+        $citizenProject = $command->getCitizenProject();
```

Comment:
You cannot call this before https://github.com/EnMarche/en-marche.fr/pull/1927/files#diff-76eb4209f0717100613e396a1c32ebc2R37.